CMakeLists.txt

```diff
@@ -107,6 +107,7 @@ set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
 set(CPACK_PACKAGING_INSTALL_PREFIX "/${CPACK_PACKAGE_NAME}")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/dist/LICENSE")
 set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_PACKAGE_NAME}")
+set(CPACK_SOURCE_IGNORE_FILES "/build/;/docker/;/packer/;/\.git/;/\.idea/;/\.DS_Store/")
```

Comment:
Use `list(APPEND CPACK_SOURCE_IGNORE_FILES somefile)` to append to list rather than replacing the build it values with your values.

Reply:
We were able to use .gitignore as a starting point, I think it worked out well.